### DISCUSSION TITLE
Versioned documentation

### DISCUSSION THREAD

**Discussion by mtazzari** [quarto-dev/quarto-cli#5342] [Q&A] [answered]
Is it possible to have versioned documentation?

For example, readthedocs automatically detects release tags from the repo and it adds a popup menu to select which version of the documentation to read. See attached pic.
readthedocs docs on versioned documentation: https://docs.readthedocs.io/en/stable/versions.html#versioned-documentation

I can't find anything on this feature in the quarto docs, but it would be really useful in many ways, e.g., to distinguish releases docs from the cutting-edge docs on the develop branch, etc.

Thanks!

![image](https://user-images.githubusercontent.com/6020226/235113576-4068bcb6-efed-41ea-95f9-5d31e739d6c6.png)

**Comment by eitsupi** [comment, marked as answer]
I think this feature request is tracked at #474.

**Reply by mcanouil** [reply]
I am closing this discussion in favour of the opened issue.
